README.md

```diff
@@ -2,6 +2,18 @@
 
 whitehall is a Ruby on Rails content management application for content published by government departments and agencies.
 
+## Running the Application
+
+Startup [using govuk-docker](https://github.com/alphagov/govuk-docker). Once running, `Whitehall` does not have an index, some suggested starting pages are below:
+
+- [`http://whitehall-admin.dev.gov.uk/government/admin/news/new`](http://whitehall-admin.dev.gov.uk/government/admin/news/new)
+- [`http://whitehall-frontend.dev.gov.uk/government/get-involved`](http://whitehall-frontend.dev.gov.uk/government/get-involved)
+
+Some pages will need local data to display
+
+- [Setup with credentials](https://docs.publishing.service.gov.uk/manual/get-started.html)
+- [Installing local data on `Whitehall`](https://github.com/alphagov/govuk-docker/blob/master/docs/how-tos.md#how-to-replicate-data-locally)
```

Comment:
It's worth noting somewhere that:
- whitehall uses mysql
- in order to get local data you'll need permission to access AWS data, which _I think_ requires production clearance which itself requires 2 second line shifts
- maaaaybe explicitly clarifying that the 2nd point is key to getting the local data but you need 1 to do it

Reply:
Updated 😁
Ah so I've been added as a `power-user` and can access the data in this context. There might be more nuance to this, i'll note _relevant_ permissions